frontend/src/App.js

```diff
@@ -5,19 +5,33 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import {Container} from "react-bootstrap";
 
 // Routing
-import { Router, Link } from "@reach/router"
+import { Router } from "@reach/router"
 
 // Pages
 import Landing from "./Views/Landing";
 import Page2 from "./Views/Page2";
 import Error from "./Views/Error";
 
+// Components
+import PickupShoppingList from "./Components/PickupShoppingList/PickupShoppingList";
+import PickupTimeslot from "./Components/PickupTimeslot/PickupTimeslot";
+import PickupUserInfo from "./Components/PickupUserInfo/PickupUserInfo";
+
+// Icons
+import { faMapMarkerAlt, faClock } from "@fortawesome/free-solid-svg-icons";
```

Comment:
My thought process was in case other pages need icons we could add the ones we need to the `library` right away at this level and be able to quickly access them in the child components. 

But I can see this potentially getting bulky. I'll try modularizing it a bit